vInputDevice.cpp

```diff
@@ -286,6 +296,10 @@ static void sendKeyThread(vInputDevice *vD)
                 cout << "Invalid Ctrl option" << endl;
             }
         } else if (vD->type == POWER) {
+            if (vD->getDevMode() == true) {
```

Comment:
Why to check mode here, by default wakeup request should call

Reply:
Updated change.